requirements.txt

```diff
@@ -28,81 +28,80 @@ Flask-WTF==0.12
 funcsigs==1.0.0
 functools32==3.2.3.post2
 future==0.15.2
-gitdb2==2.0.2
-GitPython==2.1.5
+gitdb2==2.0.3
+GitPython==2.1.9
 gunicorn==19.3.0
-html5lib==0.9999999
-hyperlink==17.3.1
+hyperlink==18.0.0
 idna==2.6
 incremental==17.5.0
-ipaddress==1.0.18
-ipython==5.4.1
+ipaddress==1.0.22
+ipython==5.6.0
 ipython-genutils==0.2.0
 itsdangerous==0.24
 Jinja2==2.8.1
+jsonschema==2.6.0
+jupyter-core==4.4.0
+kiwisolver==1.0.1
 lockfile==0.12.2
 lxml==3.8.0
 Mako==1.0.7
-Markdown==2.6.9
+Markdown==2.6.11
 MarkupSafe==1.0
-matplotlib==2.0.2
-mock==2.0.0
-nltk==3.2.4
-numpy==1.13.1
+matplotlib==2.2.2
+nbformat==4.4.0
+nltk==3.2.5
+numpy==1.14.2
 ordereddict==1.1
-pandas==0.20.3
-parsel==1.2.0
-pathlib2==2.3.0
-patsy==0.4.1
-pbr==3.1.1
-pexpect==4.2.1
+pandas==0.22.0
+parsel==1.4.0
+pathlib2==2.3.2
+patsy==0.5.0
+pexpect==4.5.0
 pickleshare==0.7.4
-ply==3.10
+plotly==2.5.1
+ply==3.11
 prompt-toolkit==1.0.15
-protobuf==3.4.0
 psutil==4.4.2
 ptyprocess==0.5.2
-pyasn1==0.3.4
-pyasn1-modules==0.1.4
+pyasn1==0.4.2
+pyasn1-modules==0.2.1
 pycparser==2.18
 PyDispatcher==2.0.5
 Pygments==2.2.0
-pyOpenSSL==17.2.0
+pyOpenSSL==17.5.0
 pyparsing==2.2.0
 python-daemon==2.1.2
-python-dateutil==2.6.1
+python-dateutil==2.7.2
 python-editor==1.0.3
 python-nvd3==0.14.2
 python-slugify==1.1.4
-pytz==2017.2
+pytz==2018.4
 PyYAML==3.12
-queuelib==1.4.2
+queuelib==1.5.0
 requests==2.18.4
-scandir==1.5
-scikit-learn==0.19.0
-scipy==0.19.1
-Scrapy==1.4.0
+scandir==1.7
+scikit-learn==0.19.1
+scipy==1.0.1
+Scrapy==1.5.0
 service-identity==17.0.0
 setproctitle==1.1.10
 simplegeneric==0.8.1
-six==1.10.0
+six==1.11.0
 slimit==0.8.1
 smmap2==2.0.3
-SQLAlchemy==1.1.14
+SQLAlchemy==1.2.7
 statsmodels==0.8.0
 subprocess32==3.2.7
 tabulate==0.7.7
-tensorflow==1.3.0
-tensorflow-tensorboard==0.1.7
 thrift==0.9.3
 traitlets==4.3.2
-Twisted==17.5.0
-Unidecode==0.4.21
+Twisted==17.9.0
+Unidecode==1.0.22
 urllib3==1.22
 vaderSentiment==2.5
-w3lib==1.18.0
+w3lib==1.19.0
 wcwidth==0.1.7
-Werkzeug==0.12.2
+Werkzeug==0.14.1
 WTForms==2.1
 zope.deprecation==4.3.0
-zope.interface==4.4.2
+zope.interface==4.5.0
```

Comment:
I still have some problems when running the `setup.sh`. Somehow the versions are not corrected. I am double checking this.